src/extension.ts

```diff
@@ -53,9 +53,16 @@ function insertAutoCloseTag(event: vscode.TextDocumentChangeEvent): void {
         if (last2chars === "</") {
             let closeTag = getCloseTag(text, excludedTags);
             if (closeTag) {
+                let nextChar = getNextChar(editor, originalPosition);
+                if(nextChar === ">")
+                    closeTag = closeTag.substr(0, closeTag.length - 1);
                 editor.edit((editBuilder) => {
                     editBuilder.insert(originalPosition, closeTag);
                 });
+                if(nextChar === ">")
+                    // editor.selection points to the / symbol
+                    // We count / + tag + >  =  closeTag.length+2
+                    editor.selection = moveSelectionRight(editor.selection, closeTag.length +2);
```

Comment:
You may change selection using Promise like https://github.com/formulahendry/vscode-auto-close-tag/blob/master/src/extension.ts#L73. And you may also use `(originalPosition, closeTag.length +1)`